doc/cabal-project-description-file.rst

```diff
@@ -85,6 +85,13 @@ architecture and version information from, which will force some
 commands (update, sdist) to require ghc present where otherwise it
 would not be necessitated.
 
+One use case for imports is to specify a `Stackage <https://www.stackage.org/>`
+snapshot, so that your cabal project can use the same set of packages as
+that snapshot. To use the ``lts-21.25`` resolver, you can write
```

Comment:
Could you link to the [cookbook reproducible builds](https://cabal.readthedocs.io/en/latest/nix-local-build.html#how-can-i-have-a-reproducible-set-of-versions-for-my-dependencies) section that already mentions importing from stackage?

Reply:
@Kleidukos sorry to ping you about this, do you know of a way to link to a different RST document without making it full link?

Reply:
Oh dear. I don't think I remember. :/